src/image-copy/azext_imagecopy/custom.py

```diff
@@ -131,11 +131,16 @@ def imagecopy(source_resource_group_name, source_object_name, target_location,
 
         tasks = []
         for location in target_location:
+            # Appending location to target name if multiple locations
+            final_target_name = target_name
+            if target_name and target_locations_count > 1:
+                final_target_name = target_name + location
```

Comment:
Let's be consistent with how it's done in the create_target method

```suggestion
                final_target_name = target_name + '-' + location
```

Reply:
I've do in that way in first time, but what if user would like to name image like that ImageNameeastus2 instead of appending '-'
Just providing more flexibility.

Reply:
Since we're adding the location to the name automatically, it's up to us to decide how we want the name to be formatted, it's not the user's decision... 
In the other method, that decision has been made already, and I think we should be consistent.

Reply:
Also consider that if the user doesn't provide a target_name, then even before your change the name of the image in each local will be something like:
xyz123-useast2

Reply:
Ok, that seems legit to me.